### DISCUSSION TITLE
How fetch updated data from CMS when using getStaticProps

### DISCUSSION THREAD

**Discussion by imkarmadev** [vercel/next.js#12577] [Help] [answered]
Hi, I implement preview mode over Prismic CMS as in tutorial I use getStaticProps. But now when I update some content on CMS, content on site still old. So I should use getServerSideProps? if yes how implement preview mode with getServerSideProps? Thx

**Comment by imkarmadev** [comment, marked as answer]
I found in the end of doc 

Works with getServerSideProps
The preview mode works on getServerSideProps as well. It will also be available on the context object containing preview and previewData.

thx, and sorry for noize

**Comment by Vadorequest** [comment]
@imkarmadev Check out https://github.com/UnlyEd/next-right-now/pull/42 and more specifically

- https://github.com/UnlyEd/next-right-now/blob/ea28688ae9/src/pages/%5Blocale%5D/index.tsx
- https://github.com/UnlyEd/next-right-now/blob/ea28688ae9b618852f69122807fb9e600d35bcca/src/utils/SSG.ts

**Reply by Vadorequest** [reply]
We don't support preview mode at this moment, gonna happen soon, maybe tomorrow
